codegen/facelift/templates/StructQObjectWrapper.template.cpp

```diff
@@ -48,7 +48,7 @@ void {{struct.name}}QObjectWrapper::init()
     QObject::connect(this, &{{struct.name}}QObjectWrapper::{{field.name}}Changed, this, &{{struct.name}}QObjectWrapper::anyFieldChanged);
     {% endfor %}
     {% if struct.isQObjectWrapperDeprecated %}
-    qCritical() << "QFace definition of structure '{{struct.qualified_name}}' is missing '@qml-component: true' annotation. This type will not be created anymore in the next version of Facelift.";
+    qCCritical(LogGenerator) << "QFace definition of structure '{{struct.qualified_name}}' is missing '@qml-component: true' annotation. This type will not be created anymore in the next version of Facelift.";
```

Comment:
A bit unrelated, but if you are at it: could you wrap this very long line, please? 120 chars should be the limit.
    "bla"
    "bla"
Maybe this should not be categorized at all (use plain qCritical), because it is so important.
This would be the only occurrence for LogGenerator, so maybe we should skip this and use different categories, like:
Q_LOGGING_CATEGORY(LogGeneral, "facelift.general")
Q_LOGGING_CATEGORY(LogModel, "facelift.model")    // everything in src/model and in  codegen/facelift/templates
Q_LOGGING_CATEGORY(LogTools, "facelift.tools")    // everything in src/desktop-dev-tools
Q_LOGGING_CATEGORY(LogIpc, "facelift.ipc")